_includes/commands/toc-cmds.md

```diff
@@ -0,0 +1,12 @@
+
+| Command      | Description                                            |
+| ------------ | ------------------------------------------------------ |
+| mceInsertToc | Inserts a Table of Contents into the editor.           |
+| mceUpdateToc | Updates the Table of Contents, if found in the editor. |
```

Comment:
Should there be a comma after `if found`? eg: `Updates the Table of Contents, if found, in the editor.`

Reply:
Replaced description with: Updates an existing Table of Contents.